### PR TITLE
Add nested STV serialization for TE

### LINKED ISSUE #114: Fetch list of STVs
For the user panel, we'll need a new endpoint to get a list of STVs without geometry for specific TE.
And another endpoint that returns shapefile of geoJSON for selected STV with an option to also return direct political relations.

### PR DESCRIPTION
**Pull request by whirish**
Fixes #114 
```js
// /api/territorial-entities/
[
    {
        "id": 1,
        "stvs": [
            {
                "id": 1,
                "start_date": "1.0",
                "end_date": "5.0",
                "territory": {
                    "type": "Polygon",
                    "coordinates": [
                        [
                            [
                                -3.955078125,
                                -4.127285323245369
                            ],
                            [
                                3.515625,
                                2.284550660236957
                            ],
                            [
                                6.6796875,
                                -7.01366792756664
                            ],
                            [
                                -3.955078125,
                                -4.127285323245369
                            ]
                        ]
                    ]
                },
                "references": [
                    "test"
                ],
                "visual_center": {
                    "type": "Point",
                    "coordinates": [
                        -0.0714111328125,
                        -0.054931632209672
                    ]
                },
                "entity": 1,
                "related_events": []
            }
        ],
        "wikidata_id": 1,
        "color": "#fff",
        "admin_level": 3,
        "relations": []
    }
]
```

### CODE REVIEW

File: project/api/serializers.py [hunk 1]
```diff
@@ -90,6 +80,18 @@ class Meta:
...
+class TerritorialEntitySerializer(ModelSerializer):
+    """
+    Serializes the TerritorialEntity model
+    """
+
+    stvs = SpacetimeVolumeSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = TerritorialEntity
+        fields = "__all__"
```
Comment (MiklerGM) [inline project/api/serializers.py]: Can we exclude `territory`? I would prefer not to fetch 400Mb of data.

**Comment by whirish** [comment]
@MiklerGM Geometry can be returned from the API, maybe we could add a query param to join with DPRs also?

**Comment by MiklerGM** [comment]
Let's merge this one. We can discuss params for geometry with DPRs in a separate issue

**Pull request merged**